__tests__/api/app.test.js

```diff
@@ -252,5 +253,72 @@ describe('Testing app', () => {
           });
       });
     });
+
+    describe('Method POST', () => {
+      test('GET: 201 status responds wih the posted comment', () => {
+        return request(app)
+          .post('/api/articles/2/comments')
+          .send({ username: 'rogersop', body: 'comment test' })
+          .expect(201);
+      });
+
+      test('GET: 201 status', () => {
+        return request(app)
+          .post('/api/articles/2/comments')
+          .send({ username: 'rogersop', body: 'comment test' })
+          .then(({ body }) => {
+            const { comment } = body;
+
+            expect(comment).toHaveProperty('comment_id', 19);
+            expect(comment).toHaveProperty('body', 'comment test');
+            expect(comment).toHaveProperty('article_id', 2);
+            expect(comment).toHaveProperty('author', 'rogersop');
+            expect(comment).toHaveProperty('votes', 0);
+            expect(comment).toHaveProperty('created_at');
+            expect(comment).toHaveProperty('created_at');
+            expect(new Date(comment.created_at)).toBeDate();
+          });
```

Comment:
There is a jest matcher _.toMatchObject_ that could clean this test up and save you from writing out _.toHaveProperty_ so many times

Reply:
done